src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -518,11 +520,26 @@ namespace DataZoneEquipment {
 
 				ZoneEquipList( ControlledZoneNum ).Name = AlphArray( 1 );
 
+				if (! lAlphaBlanks( 2 ) ){
```

Comment:
This section looks fine, although `(! lAlphaBlanks( 2 ) )`  ->  `( !lAlphaBlanks( 2 ) )`

Reply:
Fixed spacing.